### DISCUSSION TITLE
remoteDelete

### DISCUSSION THREAD

**Discussion by lewtopia** [bbernhard/signal-cli-rest-api#557] [Q&A]
The signal-cli from asamk does have a "remoteDelete" command for the sender to remote delete a message for a recipient by way of indicating the timestamp of the message.

Is there a way for this to be incorporated into this signal-cli-rest-api ?

Yes, I know I can set timers on messages to auto-delete.  However, the timer does not start until the recipient reads the message.  Ideally I would prefer to have the timer start when Home Assistant Signal Add-On sends the message, and then auto-delete/remote-Delete after the desired time has elapsed.

Unless I've missed something in the api calls?

**Comment by bbernhard** [comment]
It is not yet implemented, but I've created an issue for that.

**Reply by sepbx** [reply]
Any update on adding this?
